gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/SchemaVersionLockFileTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.dist;
+
+import com.google.common.collect.ImmutableList;
+import com.palantir.logsafe.exceptions.SafeIllegalArgumentException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class SchemaVersionLockFileTest {
```

Comment:
Can you write these as Spock tests? I believe all other tests in the repo use it.

Reply:
I thought we wanted to minimize groovy usage. I can't tell if java is supported by spock. The way this test is written is very typical for all our java tests.